src/cephadm/cephadm

```diff
@@ -2980,7 +2980,17 @@ def get_image_info_from_inspect(out, image):
 
 
 ##################################
-
+def check_subnet(subnet:str) -> Tuple[int, int, str]:
+    """Determine whether the given string is a valid subnet"""
+    version = 0
+    # ensure the format of the string is as expected address/netmask
+    if not re.search(r'\/\d+$', subnet):
```

Comment:
considering that `cluster_network` could also be a comma separated list then this won't match invalid value like `192.168.42.0,192.168.43.0/24`

Reply:
latest patch uses versions to determine whether to set bind for ipv6